crates/router/src/connector/trustpay/transformers.rs

```diff
@@ -1637,16 +1647,19 @@ pub struct Errors {
 }
 
 #[derive(Default, Debug, Serialize, Deserialize, PartialEq)]
+#[serde(rename_all = "camelCase")]
 pub struct TrustpayErrorResponse {
     pub status: i64,
     pub description: Option<String>,
     pub errors: Option<Vec<Errors>>,
+    pub instance_id: Option<String>,
```

Comment:
We shouldn't get `instance_id` for bank redirects failures right?

Reply:
yes `instance_id ` is not provided during all error scenarios.